lib/js/stanford_events.behavior.js

```diff
@@ -21,6 +21,40 @@ export default {
           $(this).attr('aria-expanded', 'false');
         }
       });
+
+      // Get today's date
+      var today = new Date();
+
+      // Update so today is all of the day
+      var a = today.setHours(0, 0, 0, 0);
+      var todayBegin = parseInt(a, 10);
+
+      // Location for label
+      const eventType =
```

Comment:
Does this work on the list displays as well?

Reply:
No it isn't work on the list display.

Reply:
ok, just curious.